csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -3476,46 +3484,148 @@ Example which fails the test:
+  * all elements required by the profile "Generic CSAF".
+  * `/document/notes` with at least one item which has a `category` of `description`, `details`, `general` or `summary`
+    > Without at least one note item with information about the "issue" which is the topic of the advisory it is useless.
+  * `/document/references`
+    > There should be a `reference` (link) to a document or website which provides more details about the issue or its remediation (if possible). This could be a link to a hardening guide, a manual, best practices or any other helpful information.
+* The value of `/document/category` SHALL be `informational_advisory`.
+* The element `/vulnerabilities` MUST NOT exist. If there is any information that would reside in the element `/vulnerabilities` the CSAF document SHOULD use another profile, e.g. "Security Advisory".
+
+If the `/product_tree` exists, a user MUST assume that all products mentioned are affected.
+
+## 5.4 Profile 4: Security Advisory
+
+This profile SHOULD be used to provide information which are related to vulnerabilities and corresponding remediations.
+
+A CSAF document SHALL fulfill the following requirements to satisfy the profile "Security Advisory":
+
+* The following elements must exist and be valid:
+  * all elements required by the profile "Generic CSAF".
+  * `/product_tree` which lists all products referenced later on in the CSAF document regardless of their state.
+  * `/vulnerabilities[]/notes`
+    > Provides details about the vulnerability.
+  * `/vulnerabilities[]/product_status`
+    > Lists for each product its status in regard to the vulnerability.
+* The value of `/document/category` SHALL be `security_advisory`.
+
+## 5.5 Profile 5: VEX
+
+This profile SHOULD be used to provide information of the "Vulnerability Exploitability eXchange". The main purpose of the VEX format is to state that and why a certain product is not affected. See [VEX] for details.
+
+A CSAF document SHALL fulfill the following requirements to satisfy the profile "VEX":
+
+* The following elements must exist and be valid:
+  * all elements required by the profile "Generic CSAF".
+  * `/product_tree` which lists all products referenced later on in the CSAF document regardless of their state.
+  * at least one of
+    * `/vulnerabilities[]/product_status/fixed`
+    * `/vulnerabilities[]/product_status/known_affected`
+    * `/vulnerabilities[]/product_status/known_not_affected`
+    * `/vulnerabilities[]/product_status/under_investigation`
+  * at least one of
+    * `/vulnerabilities[]/cve`
+    * `/vulnerabilities[]/id`
+  * `/vulnerabilities[]/notes`
+    > Provides details about the vulnerability.
+* For each item in
+  * `/vulnerabilities[]/product_status/known_not_affected` an impact statement SHALL exist in `/vulnerabilities[]/threats`. The `category` value for such a statement MUST be `impact` and the `details` field SHALL contain a a description why the vulnerability cannot be exploited.
```

Comment:
The VEX profile is in parts more strict than a security advisory, e.g. we require the use of `/vulnerabilites[]/threats` here. 
Is this intentional?

Reply:
That is a requirement of the VEX subgroup as they require impact statements. Every VEX document is a valid Security Advisory - but not every Security Advisory is a VEX document...